tests/lava/proc/plateau/test_models.py

```diff
@@ -0,0 +1,178 @@
+        expected_spk_data[10, 1] = 1
+        self.assertTrue(np.all(expected_spk_data == test_spk_data))
+
+    def test_up_dur(self):
+        """
+        Tests that the UP state lasts for the time specified by the model.
+        Checks that up_state decreases by one each time step after activation.
+        """
+        shape = (1,)
+        num_steps = 10
+        spikes_in_dend = [[3]]
+        sg_dend = SpikeGen(spikes_in=spikes_in_dend, runtime=num_steps)
+        dense_dend = Dense(weights=2 * (np.diag(np.ones(shape=shape))))
+        plat = Plateau(
+            shape=shape,
+            dv_dend=4096,
+            dv_soma=4096,
+            vth_soma=1,
+            vth_dend=1,
+            up_dur=5
+        )
+        sg_dend.s_out.connect(dense_dend.s_in)
+        dense_dend.a_out.connect(plat.a_dend_in)
+        # run model
+        test_up_state = []
+        for t in range(num_steps):
+            plat.run(RunSteps(1), Loihi2SimCfg(select_tag='fixed_pt'))
+            test_up_state.append(plat.up_state.get().astype(int)[0])
+        plat.stop()
+        # Gold standard for the test
+        # UP state active time steps 4 - 9 (5 timesteps)
+        # this is delayed by one b.c. of the Dense process
+        expected_up_state = [0, 0, 0, 5, 4, 3, 2, 1, 0, 0]
+        self.assertListEqual(expected_up_state, test_up_state)
+
+    def test_fixed_dvs(self):
+        """
+        Tests fixed point Plateau voltage decays.
+        """
+        shape = (1,)
+        num_steps = 10
+        spikes_in = [[1]]
+        sg_dend = SpikeGen(spikes_in=spikes_in, runtime=num_steps)
+        sg_soma = SpikeGen(spikes_in=spikes_in, runtime=num_steps)
+        dense_dend = Dense(weights=100 * np.diag(np.ones(shape=shape)))
+        dense_soma = Dense(weights=100 * np.diag(np.ones(shape=shape)))
+        plat = Plateau(
+            shape=shape,
+            dv_dend=2048,
+            dv_soma=1024,
+            vth_soma=100,
+            vth_dend=100,
+            up_dur=10
+        )
+        sg_dend.s_out.connect(dense_dend.s_in)
+        sg_soma.s_out.connect(dense_soma.s_in)
+        dense_dend.a_out.connect(plat.a_dend_in)
+        dense_soma.a_out.connect(plat.a_soma_in)
+        # run model
+        test_v_dend = []
+        test_v_soma = []
+        for t in range(num_steps):
+            plat.run(RunSteps(1), Loihi2SimCfg(select_tag='fixed_pt'))
+            test_v_dend.append(plat.v_dend.get().astype(int)[0])
+            test_v_soma.append(plat.v_soma.get().astype(int)[0])
+        plat.stop()
+        # Gold standard for the test
+        # 100<<6 = 6400 -- initial value at time step 2
+        expected_v_dend = [
+            0, 6400, 3200, 1600, 800, 400, 200, 100, 50, 25
+        ]
+        expected_v_soma = [
+            0, 6400, 4800, 3600, 2700, 2025, 1518, 1138, 853, 639
+        ]
+        self.assertListEqual(expected_v_dend, test_v_dend)
+        self.assertListEqual(expected_v_soma, test_v_soma)
```

Comment:
Could you check if lintin passes?
'flakeheaven lint src/lava tests'

My guess is that there are a few points that must change, including missing lines at the end of files. Not functionally relevant, but important to keep a clean code base :-)

Reply:
I've run the linting, (flakeheaven and bandit) and they pass on my local copy of the code. Also, I have the empty line at the end of the files locally, but it doesn't seem to show up on the github versions. Does github just not show the empty line at the end?